src/routes/blog/[[page]]/+page.svelte

```diff
@@ -316,4 +468,113 @@
         cursor: pointer;
         pointer-events: initial;
     }
+
+    .web-interactive-tag {
+        white-space: nowrap;
+    }
+
+    .is-selected {
+        color: hsl(var(--web-color-black));
+        background: hsl(var(--web-color-white));
+    }
+
+    .overflow-x-auto {
+        scrollbar-width: none;
+        -ms-overflow-style: none;
+    }
+
+    .overflow-x-auto::-webkit-scrollbar {
+        display: none;
+    }
+
+    .categories-wrapper {
+        position: relative;
+
+        &::before,
+        &::after {
+            content: '';
+            position: absolute;
+            top: 0;
+            width: 60px;
+            height: 100%;
+            transition: ease 250ms;
+            z-index: 100;
+            pointer-events: none;
+        }
+
+        &::before {
+            left: 0;
+            background: linear-gradient(
+                to right,
+                hsl(var(--web-color-background-docs)),
+                transparent
+            );
+        }
+
+        &[data-state='start']::before {
+            opacity: 0;
+        }
+
+        &::after {
+            right: 0;
+            background: linear-gradient(
+                to left,
+                hsl(var(--web-color-background-docs)),
+                transparent
+            );
+        }
+
+        &[data-state='end']::after {
+            opacity: 0;
+        }
+    }
+
+    .categories-wrapper {
+        flex-grow: 1;
+        max-width: 50%;
+    }
+
+    .search-and-categories > .mobile-search-bar,
+    .search-and-categories > .web-input-text-search-wrapper {
+        flex-grow: 1;
+    }
+
+    .web-input-button {
+        max-height: 40px;
+    }
+
+    @media (min-width: 1024px) {
+        .categories-wrapper {
+            flex-grow: 1;
+            max-width: 60%;
+        }
+    }
+
+    @media (min-width: 1080px) {
+        .categories-wrapper {
+            flex-grow: 1;
+            max-width: 62.5%;
+        }
+    }
+
+    @media (max-width: 768px) {
+        .search-and-categories {
+            flex-wrap: wrap;
+            display: flex !important;
+            padding-block-start: 2rem;
+            flex-direction: column-reverse !important;
```

Comment:
Are these 2 importants really needed? Can you get rid of them by making the selector more specific?

Reply:
yes, we can remove those.